src/main/java/frc/robot/scoreStates/ScoreState.java

```diff
@@ -9,9 +9,11 @@ default boolean isRed() {
         return alliance.isPresent() && alliance.get() == DriverStation.Alliance.Red;
     }
 
+    Command initializeCommand();
+
     Command driveToClosestOptimalPoint();
 
-    Command stateInitialize();
+    Command initializeSubsystem();
```

Comment:
The difference between both is not clear. Also, for example, `ShootState`'s `initializeSubsystem` uses more than one subsystem, so the name should be changed.

Reply:
I agree the that it's confusing, do you have a better name in mind?
I'll change it to subsystems.

Reply:
123fd7cc8ca85fcb6ce95024bc120c64dce7c365

Reply:
I'm not sure I fully understand the difference between them. Is it that the first "prepares" the robot to score, and the second one executes the score function? Or maybe the first function should only calculate stuff and not change things in the robot?

Reply:
The initCommand only calculates & the initSubsystem moves stuff.

Reply:
> I agree the that it's confusing, do you have a better name in mind?

> The initCommand only calculates & the initSubsystem moves stuff.

In that case, `initializeCommand()` can be `calculateTargets()`, `initializeParameters()` or something similar. `initializeSubsystems()` could be renamed to `prepareSubsystems()`. `score()` is a good name.

Reply:
6571471